File: Android/AndesiteV9/src/com/example/andesitev9/Potentiometer.java
package com.example.andesitev9;

public class Potentiometer {
	private float value;
	private String timeStamp;
	
	public Potentiometer(float value, String timeStamp) {
		this.value = value;
		this.timeStamp = timeStamp;
	}
	
	public float getValue() {
		return this.value;
	}
	
	public String getTimeStamp() {
		return this.timeStamp;
	}
	
	public int getNumberParameters()
	{
		return 10;
	}
}

File: Arduino/Arduino Libraries/Version 7 Code/Potentiometer/Potentiometer.cpp
#include "Arduino.h"
#include "Potentiometer.h"

Potentiometer::Potentiometer(int value, String timeStamp)
{
	_value = value;
    _timeStamp = timeStamp;
}

int Potentiometer::getValue()
{
	return _value;
}

String Potentiometer::getTimeStamp()
{
    return _timeStamp;
}